.github/workflows/esphome-parallel.yaml

```
@@ -44,7 +46,7 @@
         with:
           args: "version"
       - name: Copy stub files into configuration folder
-        if: ${{ steps.esphome-test-required.outputs.ESPHOME_CHANGES != 'false' }}
+        if: ${{ steps.esphome-test-required.outputs.ESPHOME_CHANGES != 'false' }}  # yamllint disable-line rule:line-length
```

actionlint (1.7.7)

property "esphome-test-required" is not defined in object type {}

(expression)